### DISCUSSION TITLE
will this project receive updates ?

### DISCUSSION THREAD

**Discussion by MCMattia** [rgossiaux/svelte-headlessui#141] [General]
Or is it closed ?

I really like it and I hope it will be further developed.

**Comment by ghost** [comment]
I just found this project recently and was disappointed to find it didn't have a good `Input` component, so fingers crossed I guess 🤞

**Comment by jeffpohlmeyer** [comment]
> I just found this project recently and was disappointed to find it didn't have a good `Input` component, so fingers crossed I guess 🤞

Look at https://headlessui.com/. There's not a "good `Input`" component there so don't hold your breath that one will be added here.

**Comment by SteelAlloy** [comment]
Although I greatly appreciate the project, it seems to be on a hold and I have no idea for how long.

Other alternatives are emerging, less mature, but with more idiomatic approaches to svelte.
Take https://github.com/captaincodeman/svelte-headlessui/ for example.

Let's see what we can learn from these two projects and move forward together!
